modules/withRouter.js

```diff
@@ -1,18 +1,28 @@
 import React from 'react'
 import hoistStatics from 'hoist-non-react-statics'
 import { routerShape } from './PropTypes'
+import warning from './routerWarning'
+
 
 function getDisplayName(WrappedComponent) {
   return WrappedComponent.displayName || WrappedComponent.name || 'Component'
 }
 
-export default function withRouter(WrappedComponent) {
+export default function withRouter(WrappedComponent, options) {
```

Comment:
I would instead use destructuring here. It essentially makes the code self-documenting. So, `{ withRef }` instead of `options`.